DuckDuckGo/TabViewController.swift

```diff
@@ -110,7 +113,8 @@ class TabViewController: WebViewController {
     
     fileprivate func onSiteRatingChanged() {
         delegate?.tab(self, didChangeSiteRating: siteRating)
-        contentBlockerPopover?.updateSiteRating(siteRating: siteRating!)
+        // contentBlockerPopover?.updateSiteRating(siteRating: siteRating!)
+        privacyDashboard?.updateSiteRating(siteRating!)
```

Comment:
This predates your code but while this is being edited let's protect it with a let statement rather than a force unwrap and maybe dismiss or clear the privacyDashboard for the nil case?